### PR TITLE
feat: add feature to filter moonraker backup files

### LINKED ISSUE #1277: Filter moonraker backup files
### Is your feature request related to a problem? Please describe

There's a 'Filter Klipper backup files' option but this doesn't filter moonraker's backup files, e.g. `moonraker.conf.backup`

### Describe the solution you'd like

Would be nice to be able to filter out any backup files: moonraker, klipper etc. 

### Describe alternatives you've considered

_No response_

### Additional information

fluidd 1.27

### PR DESCRIPTION
**Pull request by vajonam**
fixes #1277

- add filter for moonraker backup files.

### CODE REVIEW

File: src/components/widgets/filesystem/FileSystem.vue [hunk 1]
```diff
@@ -393,7 +393,9 @@ export default class FileSystem extends Mixins(StateMixin, FilesMixin, ServicesM
               return false
             }
             break
-
+          case 'moonraker_backup_files':
+            if (file.type === 'file' && file.filename.match(/^.moonraker.conf.bkp$/)) { return false }
```
Comment (pedrolamas) [inline src/components/widgets/filesystem/FileSystem.vue:397]: You are using a regular expression here, so "." has to be escaped like "\." as is on the klipper backup files expression just below.

Nevertheless, we do not need a regular expression here, so we can just compare the strings.

```suggestion
            if (file.type === 'file' && file.filename === 'moonraker.conf.bkp') { return false }
```

Comment (vajonam) [reply src/components/widgets/filesystem/FileSystem.vue]: the backup file stats with a "." so cleaned up your update.

Comment (vajonam) [reply src/components/widgets/filesystem/FileSystem.vue]: the file starts with `.` so the hidden file filter will catch this, but I like to see hidden files but hide backup of moonraker, so I like this PR, but will leave it upto you.

Comment (pedrolamas) [reply src/components/widgets/filesystem/FileSystem.vue]: Good one, I missed the starting "."

**Pull request merged**